### ISSUE TITLE
no code 

### ISSUE THREAD

**Issue by adamrek** [qinhongwei/deepfish-release#3]
can you give me your code website?   there is no code .

**Comment by GB1020** [comment]
Did you get the code?

**Comment by adamrek** [comment]
> Did you get the code?

no, I get no code yet. Can you help me ? I really want to learn it.

**Comment by GB1020** [comment]
😂😂l also have no code

**Comment by dingshunrong** [comment]
> > 您收到代码了吗？
> 
> 不，我还没有代码。你能帮助我吗 ？我真的很想学习。



> > Did you get the code?
> 
> no, I get no code yet. Can you help me ? I really want to learn it.

Do you have the code
